cmd/dep/ensure.go

```diff
@@ -80,24 +101,31 @@ dep ensure -override github.com/pkg/foo@^1.0.1
     a last resort, especially if your project may be imported by others.
 `
 
-func (cmd *ensureCommand) Name() string      { return "ensure" }
-func (cmd *ensureCommand) Args() string      { return "[spec...]" }
+func (cmd *ensureCommand) Name() string { return "ensure" }
+func (cmd *ensureCommand) Args() string {
+	return "[-update | -add] [-no-vendor | -vendor-only] [<spec>...]"
```

Comment:
oh yes, i missed `-dry-run`, tyty. 

`-examples`...i'm not actually sure, it's a flavor of a help command. seems like those are often omitted in arg lists like these?